sirepo/package_data/static/js/raydata.js

```diff
@@ -218,8 +218,34 @@ SIREPO.app.directive('replayPanel', function() {
             modelName: '=',
         },
         template: `
-          <button type="submit" class="btn btn-primary" data-ng-click="">Start Replay</button>
+          <button type="submit" class="btn btn-primary" data-ng-click="startReplay()">Start Replay</button>
         `,
+        controller: function(appState, errorService, panelState, raydataService, requestSender, $scope) {
+            $scope.startReplay = () => {
+                if (! (appState.models.replay.sourceCatalogName && appState.models.replay.destinationCatalogName && appState.models.replay.numScans)) {
```

Comment:
This can be removed once we don't show the button until all fields are valid.

Reply:
f69cb0460a842c98b329b6e5cbe841843fbbb762